addons/website/static/tests/mock_server/mock_models/website_visitor.js

```diff
@@ -70,7 +70,6 @@ export class WebsiteVisitor extends models.ServerModel {
                 anonymous_name: visitor_name,
                 channel_member_ids: membersToAdd,
                 channel_type: "livechat",
-                livechat_active: true,
                 livechat_operator_id: serverState.partnerId,
```

Comment:
Shouldn't this whole function be moved to website livechat?

There is no such fields in just website.